js/files.ts

```diff
@@ -93,17 +157,26 @@ export function close(rid: number): void {
 }
 
 /** The Deno abstraction for reading and writing files. */
-export class File implements Reader, Writer, Seeker, Closer {
+export class File
+  implements Reader, SyncReader, Writer, SyncWriter, Seeker, Closer {
   constructor(readonly rid: number) {}
 
   write(p: Uint8Array): Promise<number> {
     return write(this.rid, p);
   }
 
+  writeSync(p: Uint8Array): number {
+    return writeSync(this.rid, p);
+  }
+
   read(p: Uint8Array): Promise<ReadResult> {
     return read(this.rid, p);
   }
 
+  readSync(p: Uint8Array): ReadResult {
+    return readSync(this.rid, p);
+  }
+
   seek(offset: number, whence: SeekMode): Promise<void> {
     return seek(this.rid, offset, whence);
   }
```

Comment:
... and probably adding `seekSync()`? For symmetry with other sync op and for code that doesn't want to be wrapped in async functions

Reply:
Yeah, good suggestion :+1: